### DISCUSSION TITLE
need help with a stalker

### DISCUSSION THREAD

**Discussion by novendorhelp** [community/community#131191] [Codespaces]
### Select Topic Area

Question

### Body

Let me start of by saying i have no clue about none of this i made an account to get help,  I have been through 40 phones and had every email,password ect.stolen

**Comment by devpayoub** [comment]
check ur accpted cockies in your navig cz there is stolen throe the cockies and ensure your home Wi-Fi network is secured

**Comment by ebndev** [comment]
I’m sorry that you’re having to go through this, @novendorhelp. 

It sounds like you have been a victim of serious harassment and/or abuse. Though we can’t directly help with your situation, you might be able to receive assistance through [Crash Override Network](http://www.crashoverridenetwork.com/) and [RAINN](https://www.rainn.org/).
